modules/lang-painless/src/main/java/org/elasticsearch/painless/antlr/PainlessParser.java

```diff
@@ -1,13 +1,25 @@
 // ANTLR GENERATED CODE: DO NOT EDIT
 package org.elasticsearch.painless.antlr;
-import org.antlr.v4.runtime.atn.*;
```

Comment:
Similar question about generated vs intellij

Reply:
Picked up in optimize imports. I'll fix this.